src/madness/world/worldgop.cc

```diff
@@ -142,6 +142,8 @@ namespace madness {
             nrecv_prev = sum[1];
 
         };
+        // execute post-fence actions ... do we want to execute these with pool paused?
+        pre();
```

Comment:
@robertjharrison to execute user-provided and implicit (cleanup) actions safely we should pause the thread pool. Do you agree?